substrate/primitives/runtime/src/traits/transaction_extension/mod.rs

```diff
@@ -466,8 +461,9 @@ pub struct TransactionExtensionMetadata {
 }
 
 #[impl_for_tuples(1, 12)]
-impl TransactionExtensionBase for Tuple {
-	for_tuples!( where #( Tuple: TransactionExtensionBase )* );
+impl<Call: Dispatchable> TransactionExtension<Call> for Tuple {
+	for_tuples!( where #( Tuple: TransactionExtension<Call> )* );
```

Comment:
Should be removed according to [this comment](https://github.com/paritytech/polkadot-sdk/pull/3685#discussion_r1757470657) right?